addon/components/cfb-form-editor/question/validation.js

```diff
@@ -0,0 +1,51 @@
+import Component from "@ember/component";
+import { computed } from "@ember/object";
+import { inject as service } from "@ember/service";
+import { task } from "ember-concurrency";
+import layout from "../../../templates/components/cfb-form-editor/question/validation";
+import allFormatValidatorsQuery from "ember-caluma/gql/queries/all-format-validators";
+
+export default Component.extend({
+  classNames: ["format-validators"],
+  layout,
+
+  apollo: service(),
+
+  async didReceiveAttrs() {
+    this._super(...arguments);
+
+    await this.get("availableFormatValidators").perform();
+  },
+
+  availableFormatValidators: task(function*() {
+    const formatValidators = yield this.get("apollo").watchQuery(
+      { query: allFormatValidatorsQuery, fetchPolicy: "cache-and-network" },
+      "allFormatValidators.edges"
+    );
+    return formatValidators.map(edge => edge.node);
+  }).restartable(),
```

Comment:
`.drop()` might be better, since you don't really want this to run multiple times